src/mpcCoreKit.ts

```diff
@@ -73,6 +70,12 @@ export class Web3AuthMPCCoreKit implements IWeb3Auth {
 
   private _storageBaseKey = "corekit_store";
 
+  private enableLogging = false;
+
+  private ready = false;
+
+  private authLoggedIn = false;
+
   constructor(options: Web3AuthOptions) {
     if (!options.chainConfig) options.chainConfig = DEFAULT_CHAIN_CONFIG;
     if (options.chainConfig.chainNamespace !== CHAIN_NAMESPACES.EIP155) {
```

Comment:
we should follow injectable provider pattern same as web3auth otherwise sdk size will keep on increasing when we will add more chain providers.

Reply:
should we do it in another release?